gnovm/tests/files/zrealm_crossrealm17a.gno

```diff
@@ -0,0 +1,132 @@
+// PKGPATH: gno.land/r/crossrealm_test
+package crossrealm_test
+
+import (
+	crossrealm "gno.land/r/demo/tests/crossrealm"
+)
+
+type container struct {
+	name string
+	f    *fooer
+}
+
+func (container) Foo() { println("hello " + c.f.name) }
+
+type fooer struct{ name string }
+
+var c = &container{name: "local_container"}
+
+func main() {
+	ff := &fooer{name: "local_foo"}
+	c.f = ff
+	crossrealm.SetFooer(c)
```

Comment:
`&fooer{name: "local_foo"}` is not real yet, but it's ok as its containing object is attached by reference. And it'll be attached to the current realm later(not crossing realm).